src/Components/Nav/MenuBar.js

```diff
@@ -1,14 +1,15 @@
 import React, { Component } from "react";
+import { Link } from "react-router-dom";
 import Skin from "./Skin";
-import Body from "./Body";
+import BodyCare from "./BodyCare";
 import UvCare from "./UvCare";
 import MomBaby from "./MomBaby";
 import HotItem from "./HotItem";
 import "./MenuBar.scss";
 
 const arr = [
   { 1: <Skin /> },
```

Comment:
1.
```suggestion
const arr = {
  1: <Skin />,
```
뭔가 좀 어색해보이네요. 이렇게 고치는게 더 맞는 듯 합니다.
2. 사용되는 곳이 없다면 삭제해주세요.